manifests/init.pp

```diff
@@ -136,7 +136,7 @@
   $redis_slave_output_buffer_soft_limit = $redis::params::redis_slave_output_buffer_soft_limit,
   $redis_slave_output_buffer_soft_limit_max_interval = $redis::params::redis_slave_output_buffer_soft_limit_max_interval,
   $redis_snapshotting = $redis::params::redis_snapshotting,
-  $manage_config_file = $redis::params::manage_config_file
```

Comment:
I'm still not getting why they are not independent settings?   leaving us more compatible with upstream  puppet-redis? 

it still allows for selecting manage_config_file to true and restart_service_on_change to false achieving the requested goal? 

IMO we should be attempting to push as many changes upstream as possible.